roadmap.qmd

```diff
@@ -4,155 +4,213 @@ title: Roadmap
+| {{< var status.potential >}} | Course: Iterative project management and development for effective collaboration and work. |
+| {{< var status.potential >}} | Course: Introduction to building research software using Python as a solo developer in research environments. |
+| {{< var status.potential >}} | Course: Building research software in a team. |
+
+: "Wishlist" of learning-type documentation products we currently
+working on, will work on, or hope to work on.
+
+#### Goals
+
+This includes how-to guides, usage guides, and manuals.
+
+| Status | Guides or manuals |
+|-----------------------------------------------|------------------------|
+| {{< var status.wip >}} | [`sprout.seedcase-project.org`](https://sprout.seedcase-project.org/docs/guide) usage guide: Using and interacting with `seedcase-sprout`. |
+| {{< var status.planned >}} | Flower usage guide: Using and interacting with `seedcase-flower`. |
+| {{< var status.planned >}} | Propagate usage guide: Using and interacting with `seedcase-propagate`. |
+| {{< var status.planned >}} | Garden usage guide: Using and interacting with `seedcase-garden`. |
+| {{< var status.planned >}} | Sprout extensions usage guide: Installing or creating extensions for `seedcase-sprout`. |
+| {{< var status.planned >}} | Installing and setting up Seedcase software on server environments. |
+| {{< var status.planned >}} | Using Seedcase to create and manage data packages. |
+| {{< var status.planned >}} | A guided walkthrough: Team-based collaborative practices and workflows in a research environment. |
+| {{< var status.planned >}} | A guided walkthrough: Iterative and incremental project development and management in a research environment. |
+| {{< var status.planned >}} | Using DocOps workflows and infrastructure to reduce time between writing and dissemination. |
+| {{< var status.planned >}} | Building DataOps pipelines and infrastructure for improving research data quality and reduce time from collection to analysis. |
+| {{< var status.planned >}} | Improving the research software development lifecycle by integrating DevOps practices. |
+| {{< var status.potential >}} | Integrating research operations in organizations to improve quality and speed up research. |
+
+: "Wishlist" of goal-type documentation products we currently working
+on, will work on, or hope to work on.
+
+#### Information
+
+This includes reference manuals, API documentation, and technical
+specifications, as well as documentation for how we work together,
+decisions made, or any community contributing guidelines we have.
+
+| Status | Information resource |
+|-----------------------------------------------|------------------------|
+| {{< var status.wip >}} | [`seedcase-project/decisions`](https://github.com/seedcase-project/decisions): Archival records on the decisions made for tools or processes we use in the Seedcase Project. |
```

Comment:
It's under information because it is a reference/archive, meaning people might go to it only to get information. Like a function reference list or web API documentation.

Reply:
Looking at the diataxis website, they describe [references](https://diataxis.fr/reference/) (a.k.a. information) as e.g., : 
> In the case of software, reference guides describe the software itself - APIs, classes, functions and so on - and how to use them.

While they describe [explanations](https://diataxis.fr/explanation/) as e.g.,: 
> Explanation deepens and broadens the reader’s understanding of a subject. It brings clarity, light and context.
> Explanation is characterised by its distance from the active concerns of the practitioner. It doesn’t have direct implications for what they do, or for their work
> In the case of explanation, it’s useful to have a real or imagined *why* question to serve as a prompt. 

So, I would still think the decision posts should be under explanation? ☺️

Reply:
But, having said that, they could be placed under "explanations" if we modified them because as they currently are, they don't really fit under it. They are very technical, use jargon without explaining it, and don't provide much in terms of reflections and more explanations on the "benefits and drawbacks". They also do have some bit of "explanation" currently, so aren't fully reference... So maybe we can discuss this and see what we all want and decide how we can revise them to fit what we want.